src/blocks/mrc_call_python_function.ts

```diff
@@ -138,6 +153,12 @@ const CALL_PYTHON_FUNCTION = {
           tooltip = 'Calls the method ' + functionName + '.';
           break;
         }
+        case FunctionKind.INSTANCE_COMPONENT: {
+          const className = this.mrcComponentClassName;
+          const functionName = this.getFieldValue(pythonUtils.FIELD_FUNCTION_NAME);
+          tooltip = 'Calls the function ' + className + '.' + functionName + '.';
```

Comment:
Should the tooltip show the component name instead of the class name?

Reply:
Yes. Actually I think it should show both, since the name of the component might not overlap with the component class name.